content/project/about/contents+pt.lr

```diff
@@ -0,0 +1,131 @@
+body:
+
+História
+=======
```

Comment:
I'm seeing some warnings due to the "title underline" thing not being aligned with the title text, for this and for other titles.
Could you fix those?